### PR TITLE
fix docs release type

### PR DESCRIPTION
**Pull request by apahim**
It is expected to be string, not integer.

Signed-off-by: Amador Pahim <apahim@redhat.com>

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -98,7 +98,7 @@
 copyright = u'2014-2015, Red Hat'
 
 version = VERSION
-release = 0
+release = '0'
```
Comment (clebergnu) [inline docs/source/conf.py]: Nice catch!  Although I did not see any error coming out of the docs build, did you?

Comment (apahim) [inline docs/source/conf.py]: Yep...
~~~
======================================================================
ERROR: test_build_docs (__main__.DocBuildTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/apahim/src/avocado/selftests/doc/test_doc_build.py", line 77, in test_build_docs
    raise DocBuildError(e_msg)
DocBuildError: 1 ERRORS and/or WARNINGS detected while building the html docs:
1) WARNING: The config value `release' has type `int', defaults to `str.'
~~~

Comment (clebergnu) [inline docs/source/conf.py]: That is weird... probably related to sphinx versions... on my system:

```
$ python -m unittest discover selftests.doc 
.
----------------------------------------------------------------------
Ran 1 test in 14.878s

OK
```

Anyway, the fix looks good. Thanks!

Comment (apahim) [inline docs/source/conf.py]: Sphinx (1.4b1)

**Pull request merged**